src/app/content/components/Page/scrollToTopOrHashManager.ts

```diff
@@ -58,7 +58,7 @@ const scrollToTopOrHashManager = (
   ) {
     return;
   }
-  if (previous.page !== current.page) {
+  if (!previous || (previous.page !== current.page)) {
```

Comment:
unnecessary parens, and i think this can be written like this right?
```suggestion
  if (previous.?page !== current.page) {
```